Library/Homebrew/cask/lib/hbc/exceptions.rb

```diff
@@ -29,13 +29,30 @@ def to_s
 
   class CaskAlreadyInstalledError < AbstractCaskErrorWithToken
     def to_s
-      %Q{A Cask for #{token} is already installed. Add the "--force" option to force re-install.}
+      s = <<-EOS.undent
+        A Cask for #{token} is already installed.
+      EOS
+
+      s.concat("\n").concat(reinstall_message)
+    end
+
+    private
+
+    def reinstall_message
+      <<-EOS.undent
+        To re-install #{token}, run:
+          brew cask uninstall --force #{token}; brew cask install #{token}
```

Comment:
I’d say `&&` makes more sense. If it were `brew cask uninstall --force #{token}; brew cask install --force #{token}` ( `--force` also in `install`), `;` would suffice (as a kind of “it’ll work one way or another”, but as is, we don’t want the user to be stuck in a loop of it always showing the same message (because it could not install). We want it to fail as soon as possible so they can file a bug.